iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuiz/Views/StepQuizView.swift

```diff
@@ -257,15 +264,15 @@ struct StepQuizView: View {
                 break
             }
         case .showProblemsLimitReachedModal:
-            #warning("TODO: implement showing problems limit reached modal")
+            presentProblemsLimitReachedModal()
         case .showProblemOfDaySolvedModal(let showProblemOfDaySolvedModalViewAction):
             presentDailyStepCompletedModal(earnedGemsText: showProblemOfDaySolvedModalViewAction.earnedGemsText)
         case .navigateTo(let viewActionNavigateTo):
             switch StepQuizFeatureActionViewActionNavigateToKs(viewActionNavigateTo) {
             case .back:
                 stackRouter.popViewController()
             case .home:
-                #warning("TODO: implement navigation to home")
+                TabBarRouter(tab: .home).route()
```

Comment:
Here you just change the active tab index, but we also need to navigate back in a hierarchy